hadoop-ozone/dist/src/main/smoketest/ozonefs/setup.robot

```diff
@@ -30,6 +30,10 @@ ${DEEP_DIR}          test/${SCHEME}/dir
 
 *** Keywords ***
 Setup for FS test
+    ${random} =         Generate Random String  5  [NUMBERS]
+    Set Suite Variable  ${BUCKET}  ${random}-${BUCKET_TYPE}1-${SCHEME}
+    Set Suite Variable  ${BUCKET2}  ${random}-${BUCKET_TYPE}2-${SCHEME}
+    Set Suite Variable  ${BUCKET_IN_VOL2}  ${random}-${BUCKET_TYPE}3-${SCHEME}
```

Comment:
Nit: I think it would be better to add these to `Assign suite vars for FS test`.  If you agree, then invocation of that keyword needs to be moved earlier here.

Reply:
Good idea. Moved this block to there.